zombies.js

```diff
@@ -79,6 +95,119 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+class Player {
+  constructor(name, health, strength, speed) {
+    this.name = name;
+    this.health = health;
+    this.strength = strength;
+    this.speed = speed;
+    this.isAlive = true;
+    this.equipped = false;
+    this._pack = [];
+    this._maxHealth = 100;
+  }
+  getPack() {
+    return this._pack;
+  }
+
+  getMaxHealth() {
+    return this._maxHealth;
+  }
+
+  checkPack() {
+    let pack = this.getPack();
+    console.log(pack);
+    // does it need return before console.log?
```

Comment:
It doesn't not need a return since there's no other code below that line, and because console.log doesn't return anything.